aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/stash/StashTest.java

```diff
@@ -29,14 +29,15 @@
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 import org.jenkinsci.plugins.workflow.test.steps.SemaphoreStep;
-import org.junit.Test;
-import static org.junit.Assert.*;
 import org.junit.ClassRule;
 import org.junit.Rule;
+import org.junit.Test;
 import org.jvnet.hudson.test.BuildWatcher;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
FWIW [usual code style](https://github.com/jenkinsci/branch-api-plugin/blob/master/CONTRIBUTING.md#imports) is to use static imports for these.